src/util/errors/errors.go

```diff
@@ -0,0 +1,42 @@
+package errors
```

Comment:
If this is only used in package `cipher`, move this code to that package. I don't expect this to be used in other packages

Reply:
this is meant to be used by other packages e.g. `coin` , but those changes are being implemented in #1639 ... which will be ready to merge after adding this , btw.

Reply:
[Test results](https://travis-ci.org/skycoin/skycoin/builds/412857210) after merging with [`stdevEclipse_t1191_test_libc_cipher_and_coin`](https://github.com/skycoin/skycoin/pull/1639) . Some of those involve `coin` transactions .

`[====] Synthesis: Tested: 143 | Passing: 132 | Failing: 11 | Crashing: 6`

Reply:
Remove it then, this is unnecessary

Reply:
I forgot to mention something ... if this is moved on to cipher then there will be a circular import issue making impossible to compile `libskycoin` code .

Reply:
I think I can work around this , maybe ... question is can `coin` import `cipher` ?

Reply:
```
$ grep -nr 'cipher' src/coin/ | wc -l
     151
```

Reply:
this is feasible ... will do it this way in `cipher`.

Reply:
Just remove it, its not necessary